src/triage/experiments/singlethreaded.py

```diff
@@ -40,6 +40,8 @@ def catwalk(self):
                 ),
                 matrix_store=train_store
             )
+            # remove Nones from model_ids so we don't try to predict/evaluate on skipped baselines
+            model_ids = list(filter(None.__ne__, model_ids))
```

Comment:
Mmmmm, I don't like that . Are you trying to remove the empty `model_ids`?

Reply:
Checking the practices, It appears that in python 3 your approach is the recommended one:

https://stackoverflow.com/questions/16096754/remove-none-value-from-a-list-without-removing-the-0-value

So, I am happy with that :)

(But I still don't like it)

Reply:
yes, @jesteria also raised this. https://github.com/dssg/triage/pull/356#discussion_r159285356

Reply:
we can use a list comprehension instead, if we like:

```>>> model_ids = [0, None, 1, 2, 3]
>>> list(filter(None, model_ids))
[1, 2, 3]
>>> [model_id for model_id in model_ids if model_id is not None]
[0, 1, 2, 3]
>>> list(filter(None.__ne__, model_ids))
[0, 1, 2, 3]```